lib/phases/cold-launch.js

```diff
@@ -185,22 +160,22 @@ ColdLaunch.prototype.getAppCoordinates = function() {
 
   return marionette
     .startSession()
-    .then(function(client) {
+    .then((client) => {
       client.switchToFrame();
 
       // Capture the full height of the System to determine how much shorter
       // the homescreen is
-      var systemHeight = client.executeScript(function() {
+      let systemHeight = client.executeScript(function() {
```

Comment:
Since these functions are stringified and executed in the client, I wasn't for sure about arrow functions since they have different scoping mechanisms. For these, I think it's safer to leave as normal functions.